packages/shared/src/components/comments/CommentInputOrModal.tsx

```diff
@@ -18,38 +20,22 @@ interface CommentInputOrPageProps
   replyToCommentId?: string;
 }
 
-export default function CommentInputOrPage({
+export default function CommentInputOrModal({
   onClose,
   className,
   ...props
-}: CommentInputOrPageProps): ReactElement {
-  const isMobile = useViewSize(ViewSize.MobileL);
-  const router = useRouter();
+}: CommentInputOrModalProps): ReactElement {
+  const isModal = !useViewSize(ViewSize.Laptop);
```

Comment:
I believe we should keep the same behavior on tablet.

![image](https://github.com/dailydotdev/apps/assets/13744167/70c18e5e-adb9-46c6-a322-f92d8f31d803)

Reply:
True, it should work as today for tablet 👍 Changing...

Reply:
Fixed